packages/flutter/test/material/dialog_test.dart

```diff
@@ -217,6 +217,57 @@ void main() {
     expect(materialWidget.shape, customBorder);
   });
 
+   testWidgets('showDialog child and builder cannot be simultaneously defined', (WidgetTester tester) async {
```

Comment:
nit: indentation is off
```suggestion
  testWidgets('showDialog child and builder cannot be simultaneously defined', (WidgetTester tester) async {
```